pom.xml

```diff
@@ -27,15 +27,16 @@
     </developers>
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
-        <jenkins.version>2.440.3</jenkins.version>
+        <jenkins.version>2.452.4</jenkins.version>
         <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
+    <maven.compiler.release>17</maven.compiler.release>
     </properties>
     <dependencyManagement>
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-2.440.x</artifactId>
-                <version>3435.v238d66a_043fb_</version>
+                <artifactId>bom-2.452.x</artifactId>
```

Comment:
This should anyway be using the newer idiom with a shared property for LTS line.

Reply:
https://github.com/jenkinsci/archetypes/issues/738
https://github.com/jenkinsci/plugin-modernizer-tool/issues/368

Need most likely to be fixed on https://github.com/openrewrite/rewrite-jenkins/blob/main/src/main/java/org/openrewrite/jenkins/AddPluginsBom.java

The recipe is already handling the mismatch of jenkins.version and bom. But since we moved to the new property the recipe must be adapted